sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/DoNotDecreaseMemberVisibility.cs

```diff
@@ -100,9 +100,15 @@ public Diagnostic FindIssue(MemberDeclarationSyntax memberDeclaration)
                 var methodSymbol = memberSymbol as IMethodSymbol;
                 if (methodSymbol != null)
                 {
+                    var methodDeclaration = memberDeclaration as MethodDeclarationSyntax;
+                    if (methodDeclaration?.Modifiers.Any(m => m.ToString() == "new") == true)
```

Comment:
`== true` is not pointless, because of the conditional access operator, but frankly speaking, I prefer something like this, even if it is longer:
```
if (methodDeclaration != null &&
    methodDeclaration.Modifiers.Any(m => m.ToString() == "new")) 
```